pkg/lib/packagekit.es6

```diff
@@ -323,7 +323,7 @@ export function watchRedHatSubscription(callback) {
             .done(result => callback(validSubscriptionStates.indexOf(result[0]) >= 0))
             .fail(ex => {
                 if (ex.problem != "not-found")
-                    console.warn("Failed to query RHEL subscription status:", ex);
+                    console.warn("Failed to query RHEL subscription status:", JSON.stringify(ex));
```

Comment:
What about `ex.toString()`? That's what we'd use when showing the error in the UI.

Reply:
At least initially I prefer the full JSON form, this might help in debugging that occasional failure.